src/UI/Components/Diagram/actions.ts

```diff
@@ -642,10 +627,10 @@ function handleAttributes(
  * @param {"candidate" | "active"=} presentedAttrs *optional* indentify used set of attributes if they are taken from Service Instance
  * @returns {void}
  */
-function handleInfoIcon(
+export function addInfoIcon(
   instanceAsTable: ServiceEntityBlock,
-  presentedAttrs?: "candidate" | "active",
-) {
+  presentedAttrs: "candidate" | "active",
```

Comment:
Make ticket: there's a type for these already; it would be better to use the type using the Pick type.

Reply:
https://github.com/inmanta/web-console/issues/6013